code/datums/diseases/advance/symptoms/viral.dm

```diff
@@ -80,10 +82,10 @@ Bonus
 */
 
 /datum/symptom/viralreverse
-
 	name = "Viral Aggressive Metabolism"
 	desc = "The virus sacrifices its long term survivability to nearly instantly fully spread inside a host. \
 	The virus will start at the last stage, but will eventually decay and die off by itself."
+	symptom_flags = SYMPTOM_DEAD_TICK_WEAK
```

Comment:
The to_chat message should only happen when alive

Reply:
Viral Aggressive Metabolism + functioning Necrotic + deadly symptoms is not an interactive way to antagonize rounds at all. This is right up there with plasma flooding in terms of fun for the victims and ruining rounds. 

This symptom should not work in a dead host at all. If the virus is so deadly that it is completely flooring people before lethality fades, enabling it to work with necrotic only enables extremely unfun situations for crew.